src/UI/Component/Panel/Factory.php

```diff
@@ -79,6 +79,9 @@ public function sub($title,$content);
 	 *   rivals:
 	 *      Standard Panels: >
 	 *        The Report Panels contains sub panels used to structure information.
+	 *      Presentation Table: >
+	 *        Presentation Tables display only a subset of the data at first glance;
```

Comment:
Collapsable Tables present a subset of data which can inspected item-by-item by expanding the full set of set of data.

Reply:
One can expand multiple rows, not only one (as in the accordion). Tried to make that a little clearer in the documentation.